### DISCUSSION TITLE
Warning and critical error after upgrading to 0.24 on Linux Mint

### DISCUSSION THREAD

**Discussion by efermon** [flet-dev/flet#4097] [Q&A]
### Question

I am a retired person and I study Flet as a hobby, so I am not in any development team nor do I have any applications in production.

The issues occur after updating Flet to version 0.24.0. I have reverted to version 0.23 and the issues are gone. 

Whether you run with python or flet run, when you close the app you always get 2 warnings.

Two warnings on closing do not affect the operation of the program. But if I add the WindowDragArea component, when I drag the window two errors appear that sometimes cause erratic operation of the application:

### Code sample

```python
import flet as ft

def main(page: ft.Page):
    page.window.max_height = 520
    page.window.max_width = 400
    page.window.frameless = True
    page.add(
        ft.WindowDragArea(
           ft.Container(
              content = ft.Row(
                  [ ft.Text('WindowDragArea', color='WHITE'),
                    ft.IconButton(ft.icons.CLOSE, icon_color="WHITE", on_click=lambda _: page.window.close())
                  ],  alignment="spaceBetween" ),
              bgcolor = 'blue',
             )
           ) 
    )

ft.app(target=main)
```

### Error message

```bash
Warnings on closing:
** (flet:4241): WARNING **: 17:02:59.271: Attempted to set message handler on an FlBinaryMessenger without an engine

** (flet:4241): WARNING **: 17:02:59.271: Attempted to set message handler on an FlBinaryMessenger without an engine

Critical error when dragging:
(flet:14475): Gdk-CRITICAL **: 19:00:49.970: gdk_device_get_source: assertion 'GDK_IS_DEVICE (device)' failed
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by sonalita** [comment]
Also getting the same issue

-  flet version 0.24.1
- Python version 3.10.12
-  OS is Linux Mint 21.3 Cinnamon
- kernel is 5.15.0-91-generic

**Comment by Pshemas** [comment]
same on my end:
![obraz](https://github.com/user-attachments/assets/c004a887-2ef4-4600-bd57-a5ca5e3da05d)

- Flet 0.24.1
- Python 3.12

Runnning on OpenSUSE Tumbleweed

**Comment by iamlostshe** [comment]
![image](https://github.com/user-attachments/assets/ee64fdb8-5c8d-4d81-830c-ede34dc49fde)

![Снимок экрана от 2024-12-24 01-14-53](https://github.com/user-attachments/assets/ee43419f-fb5b-4631-b508-138dc7227270)


Python 3.12.7

**Reply by fccoelho** [reply]
I am also having this issue on flet 0.27.6 on Linux
